src/Mod/TechDraw/App/Cosmetic.cpp

```diff
@@ -344,14 +344,14 @@ CosmeticEdge::CosmeticEdge()
 {
 //    Base::Console().Message("CE::CE()\n");
     permaRadius = 0.0;
-    m_geometry = new TechDraw::BaseGeom();
+    m_geometry = std::make_shared<TechDraw::BaseGeom> (BaseGeom());
```

Comment:
The call of BaseGeom() inside the parentheses can be dropped. What it does is create an instance on the stack and afterward it invokes the default copy constructor. So, this should be sufficient: `m_geometry = std::make_shared<TechDraw::BaseGeom> ();`